tests/test_tpp_backend.py

```diff
@@ -124,3 +129,115 @@ class Cohort:
     assert extract(Cohort, TPPBackend, database) == [
         dict(patient_id=1, admission=date(2020, 12, 12))
     ]
+
+
+@pytest.mark.integration
+def test_hospitalization_table_returns_a_single_code(database, setup_tpp_database):
```

Comment:
These tests look like they could easily be a single parametrized test

Reply:
Will this just work with filtering on codelists?  I think they will, but maybe a test for it would be useful

Reply:
> These tests look like they could easily be a single parametrized test

You're quite right. Good idea.

Reply:
> Will this just work with filtering on codelists? I think they will, but maybe a test for it would be useful

I feel like the answer is "of course", but I can't completely convince myself it's right -- which I think is a good signal that a test is needed.